content/docs/reference-dom-elements.md

```diff
@@ -73,70 +73,70 @@ const divStyle = {
+  Bonjour, monde !
 </div>
 
-// Result style: '10%'
+// Style résultat : '10%'
 <div style={{ height: '10%' }}>
-  Hello World!
+  Bonjour, monde !
 </div>
 ```
 
-Not all style properties are converted to pixel strings though. Certain ones remain unitless (eg `zoom`, `order`, `flex`). A complete list of unitless properties can be seen [here](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59).
+Toutes les propriétés de style ne sont pas systématiquement converties en pixels pour autant. Certaines restent sans unité (e.g. `zoom`, `order`, `flex`). La liste complète des propriétés sans unité peut-être consultée [ici](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59).
 
 ### suppressContentEditableWarning {#suppresscontenteditablewarning}
 
-Normally, there is a warning when an element with children is also marked as `contentEditable`, because it won't work. This attribute suppresses that warning. Don't use this unless you are building a library like [Draft.js](https://facebook.github.io/draft-js/) that manages `contentEditable` manually.
+Normalement un avertissement apparaît lorsqu'un élément avec des enfants est également marqué comme `contentEditable` car ça ne fonctionnera pas. Cet attribut supprime cet avertissement. Ne l'utilisez pas à moins de développer une librairie comme [Draft.js](https://facebook.github.io/draft-js/) qui gère `contentEditable` manuellement.
 
 ### suppressHydrationWarning {#suppresshydrationwarning}
 
-If you use server-side React rendering, normally there is a warning when the server and the client render different content. However, in some rare cases, it is very hard or impossible to guarantee an exact match. For example, timestamps are expected to differ on the server and on the client.
+Si vous utilisez le rendu côté serveur de React, normalement un avertissement apparaît lorsque le serveur et le client ont un contenu rendu différent. Cependant, dans certains cas rares, il est très difficile, voir impossible, de garantir un contenu identique. Par exemple s'il est prévu que des timestamps diffèrent entre le serveur et le client.
 
-If you set `suppressHydrationWarning` to `true`, React will not warn you about mismatches in the attributes and the content of that element. It only works one level deep, and is intended to be used as an escape hatch. Don't overuse it. You can read more about hydration in the [`ReactDOM.hydrate()` documentation](/docs/react-dom.html#hydrate).
+Si vous définissez `suppressHydrationWarning` à `true`, React ne vous avertira pas des différences dans les attributs et le contenu de cet élèment. Ça ne fonctionne que pour un niveau de profondeur, et est prévu comme une échappatoire. N'en abusez pas. Vous pouvez en apprendre plus au sujet de la phase d'hydratation dans la [`documentation de ReactDOM.hydrate()`](/docs/react-dom.html#hydrate).
```

Comment:
```suggestion
Si vous définissez `suppressHydrationWarning` à `true`, React ne vous avertira pas des différences dans les attributs et le contenu de cet élément. Ça ne fonctionne qu’à un niveau de profondeur, et c’est conçu comme une échappatoire. N'en abusez pas. Pour en apprendre davantage sur la phase d'hydratation, consultez la [`documentation de ReactDOM.hydrate()`](/docs/react-dom.html#hydrate).
```

- Aaaaah on n'a jamais écrit (ni prononcé) « élèment », dis donc…  Réflexe musculaire depuis « évènement » ? 😉 
- Parfait le « n'en abusez pas » 💯 
- On ré-écrit parfois le “you can learn more” vers une forme plus active et respirante, comme ici.